### ISSUE TITLE
Identifier 'minSearchStringLength' has already been declared.

### ISSUE THREAD

**Issue by Zegnat** [adeekshith/netflix-categories#3]
As soon as I click the icon, the following error is recorded:

```
Uncaught SyntaxError: Identifier 'minSearchStringLength' has already been declared
```

If I find the time today I will look into it more, but I am short on time and wanted to document it here first.

**Comment by adeekshith** [comment]
Oh! Thats strange! Thanks for reporting @Zegnat . Will take a look.

**Comment by adeekshith** [comment]
Caused by loading [popup.js](https://github.com/adeekshith/netflix-categories/blob/9f16813efe6d2c6ce2b2507520b9e00376e620b3/webextension/browser-action/js/popup.js) multiple times in [popup.html](https://github.com/adeekshith/netflix-categories/blob/9f16813efe6d2c6ce2b2507520b9e00376e620b3/webextension/browser-action/popup.html) 

Fixed in PR #8 commit c9abed0
